### PR TITLE
Add admin custom menu

### PR DESCRIPTION
**Pull request by szok**
We have to choose which solution:
#1454  or #1453

### CODE REVIEW

**Comment by ghost** [comment]
:ok:

File: src/ralph/admin/sites.py [hunk 1]
```diff
@@ -47,86 +43,11 @@ class RalphAdminSiteMixin(object):
...
-            self.each_context(request),
-            title=self.index_title,
-        )
-        return TemplateResponse(
-            request, self.index_template, context
-        )
-
     def each_context(self, request):
-        """Override django each context method."""
+        """Override each_context admin stie method."""
```
Comment (vi4m) [inline src/ralph/admin/sites.py]: site

**Comment by szok** [comment]
screen shot 2015-05-28 at 07 08 12

**Comment by ar4s** [comment]
:ok:

**Comment by vi4m** [comment]
gr8

**Pull request closed without merging**